anthos-deployment/main.tf

```diff
@@ -0,0 +1,64 @@
+resource "google_container_cluster" "primary" {
+  provider = google-beta
+
+  name                     = var.primary_name
+  location                 = var.location
+  remove_default_node_pool = true
+  initial_node_count       = 1
+
+  release_channel {
+    channel = var.release_channel
+  }
+
+  depends_on = [google_project_service.container]
+}
+
+resource "google_container_node_pool" "primary_preemptible_nodes" {
+  provider = google-beta
```

Comment:
Just curious, which GKE's feature require the `beta` version?

Reply:
I had something,  don't recall but I removed it now.